dosma/core/med_volume.py

```diff
@@ -899,6 +955,72 @@ def is_mmap(self) -> bool:
         # is also a memmap object, but should not be symlinked or copied
         return isinstance(self.A, np.memmap) and isinstance(self.A.base, mmap)
 
+    @property
+    def is_zarr(self) -> bool:
```

Comment:
Is there a use case you were envisioning with this property? If this check can be done externally (i.e. in user code, not as part of the `MedicalVolume` class), that may be preferred to avoid setting a precedent for `is_x` methods in  medical volume

Reply:
I vaguely remember there was an `is_nib` property, apart from that there was no reason to include it. Removing it sounds good.